pages/recrutement/index.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <div>
+    <div class="fr-mb-5w">
+      <h1 class="fr-text-title-blue-france"
+
+      >Recrutement
+      </h1>
+      <p>
+        Nous accompagnons les territoires vers leur transition numérique le plus simplement
+        possible.
+      </p>
+      <nuxt-link class="it-primary" to="">
```

Comment:
c'est ok d'avoir le `to=""` vide ou c'est mieux `"/"` ? (je n'ai pas vérifié la doc là)

Reply:
Non c'est un oubli j'ajoute le lien vers la page de recrutement beta-gouv. Sinon pour repondre a la question, il me semble que la bonne valeur "vide" est le `#`

Reply:
Ah oui, ce n'est pas un lien d'accueil mais un non-lien temporaire, j'avais pas bien vu…

Reply:
Je viens de le push en balise `a` d'ailleurs pas de raison que ce soit un `nuxt-link`